app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -14,6 +14,8 @@ def user_info
       iss: root_url,
       email: email_from_sp_identity(identity),
       email_verified: true,
+      alternate_emails: alternate_emails_from_sp_identity(identity),
+      alternate_emails_verified: true,
```

Comment:
I would drop this attribute... the only reason we have `email_verified` is it's part of the spec and we kept it, so since we're inventing a new trait I would not add a corresponding `_verified`

However if we do feel strongly about keeping it, I think it should be an array (that could be zipped with the emails, or maybe even a hash/map? because a single boolean doesn't allow for only some to be verified